src/main/java/bio/terra/cli/command/resource/addref/GitRepo.java

```diff
@@ -41,6 +47,18 @@ protected void execute() {
     bio.terra.cli.businessobject.resource.GitRepo addedResource =
         bio.terra.cli.businessobject.resource.GitRepo.addReferenced(createParams.build());
     formatOption.printReturnValue(new UFGitRepo(addedResource), GitRepo::printText);
+    ExternalCredentialsManagerService ecmService = ExternalCredentialsManagerService.fromContext();
+    try {
+      ecmService.getSshKeyPair(SshKeyPairType.GITHUB);
+    } catch (SystemException e) {
+      if (e.getCause() instanceof HttpStatusCodeException
+          && ((HttpStatusCodeException) e.getCause()).getStatusCode() == HttpStatus.NOT_FOUND) {
+        throw new UserActionableException(
+            "You do not have a Terra ssh key, cloning the git repo in the GCP notebook will"
+                + " fail. Please run `terra user ssh-key generate` and store the output (public key) in"
+                + " your GitHub account https://github.com/settings/keys.");
```

Comment:
I thought instructions were at https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#adding-your-ssh-key-to-the-ssh-agent ?